web/src/avatar.js

```diff
@@ -39,6 +40,23 @@ export function build_bot_edit_widget($target) {
         return $target.find(".edit_bot_avatar_file_input");
     };
 
+    if ($("#current_bot_avatar_source").val() === "G") {
+        $("#edit-bot-avatar-upload-widget .image-hover-background").hide();
+        $("#edit-bot-avatar-upload-widget .image-delete-button").hide();
+    }
+
+    $("#edit-bot-avatar-upload-widget .image-delete-button").on("click", (e) => {
+        e.preventDefault();
+        e.stopPropagation();
+        $("#edit-bot-avatar-upload-widget .image-hover-background").hide();
+        $("#edit-bot-avatar-upload-widget .image-delete-button").hide();
+        $("#current_bot_avatar_source").val("G");
+        $("#current_bot_avatar_source").trigger("input");
```

Comment:
I guess this is to enable the save button if it is disabled. But maybe we can do something better, probably just enable the save button directly.